src/middlewares/signUpValidation.js

```diff
@@ -0,0 +1,19 @@
+import Joi from '@hapi/joi';
```

Comment:
The filename for a middleware should be in the format `<name>.middleware.js`. E.g. `auth.middleware.js`